.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Hydra W3C Community Group
+
+This repository is being used for work in the Hydra W3C Community Group, governed by the
+[W3C Community License Agreement (CLA)](http://www.w3.org/community/about/agreements/cla/).
+To contribute, you must [join the Community Group](http://www.hydra-cg.com/#community).
+
+## How to contribute
+
+To advance Hydra, it is imperative that the community engages with us in discussing new and existing features. The only
+way to hope for an end result useful for as many people as possible is for the actual interested parties to collaborate
```

Comment:
Can we speak about an end result? I think there may be always something new to cover :)

Reply:
You mean to add some more text here?

Reply:
No, maybe some rewording. Maybe:

> The only way to hope for the vocabulary being useful for as many people as possible ...